aldryn_faq/views.py

```diff
@@ -171,6 +221,8 @@ def get(self, request, *args, **kwargs):
         return response
 
     def get_category_url(self):
+        if not hasattr(self, 'object'):
+            self.object = self.get_object()
```

Comment:
self.object is handled by View and ultimately BaseView, neither of which declare it as a class property, but it is normally set in get(). I guess I'm doing it like this in case get_category_url() is called before get().